tests/src/JIT/Performance/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,38 @@
+@setlocal
+@echo off 
+
+set ARCH=%1
+set BUILD=%2
+
+if /I "%ARCH%" == "" set ARCH=x64
+if /I "%BUILD%" == "" set BUILD=Release
+
+rem *** set this appropriately for enlistment you are running benchmarks in
+if /I NOT "%CORECLR_ROOT%" == "" goto over
```

Comment:
I need to know the actual enlistment root base, not a relative directory path.    If there is a better way to get this but looking at current directory, I could use that.    I just don't know how to do that.